poetry/console/commands/publish.py

```diff
@@ -14,6 +16,18 @@ class PublishCommand(Command):
         ),
         option("username", "u", "The username to access the repository.", flag=False),
         option("password", "p", "The password to access the repository.", flag=False),
+        option(
+            "custom-ca",
```

Comment:
I probably struggled the most with naming the silly parameters compared to anything else about this PR.  I started with custom-ca, hated it, but then figured we could come to consensus in the PR itself :)

Since I'm interfacing with both `requests` and pip under the hood, I need to use a combined client cert/key file as that's the only thing both take (`pip` is the picky one).  That being said, I liked `pip`'s `--client-cert` and went with it.

For CA:

- Twine (as you noted) uses `--cacert`
- pip (as you noted) uses `--cert`
- cURL uses `--cacert`
- wget uses `--ca-certificate`
- `requests` uses the parameter `verify` (that's terrible)
- Python's built-in `ssl` module uses the parameter `cafile`
- `urllib3` uses the parameter `ca_certs`

Since I did `--client-cert` (with the hyphen in between), I'm thinking it should be consistent and be `--ca-<SOMETHING>` but that looks weird too.  I'm happy to change it to anything that the maintainers will accept; I fully recognize `--custom-ca` is not great.

Reply:
I think we can follow what's already used in both `pip` and `twine`, i.e. `--cert` and `--client-cert`.